src/containers/FourAndMore.tsx

```diff
@@ -40,23 +43,81 @@ const getFourAndMore = (users: { [key: string]: any }) => {
   return fourAndMore;
 };
 
-const LIMIT = 20;
+const LIMIT = 100;
 const FourAndMoreContainer: React.FC<FourAndMoreContainerProps> = (props) => {
   const { users } = props;
   const [page, setPage] = useState(0);
   const [prevUserLength, setPrevUserLength] = useState(0);
+  const [timeframe, setTimeframe] = useState('All Time');
+  const [startTime, setStartTime] = useState(0);
+  const [endTime, setEndTime] = useState(0);
   const hasMore = () => {
     return prevUserLength !== users.length;
   };
+
   useEffect(() => {
     setPrevUserLength(users.length);
   }, [users]);
+
   const loadFunc = () => {
     setPage(page + 1);
-    props.fetchLeaderboard(page * LIMIT + 3, LIMIT);
+    if (startTime === 0 || endTime === 0) {
+      props.fetchLeaderboard(page * LIMIT + 3, LIMIT);
+    } else {
+      props.fetchLeaderboard(page * LIMIT + 3, LIMIT, startTime, endTime);
+    }
   };
+
+  const timeframeData = [
+    {
+      text: 'All Time',
+      start: 0,
+      end: 0,
+    },
+    {
+      text: '2020-2021',
+      start: 1592204400, // June 15, 2020 in epoch seconds
+      end: 1623740400, // June 15, 2021 in epoch seconds
+    },
+    {
+      text: '2019-2020',
+      start: 1560582000, // June 15, 2019 in epoch seconds
+      end: 1592204400, // June 15, 2020 in epoch seconds
+    },
+  ];
```

Comment:
Wonder if we can make this dynamic for upcoming years so that we don't have to update this array in future iterations. Possibly pulling the current year -> incrementing year by 1.

Reply:
That's doable - we can have a variable for the earliest year (`earliest = 1560582000, // June 15, 2019 in epoch seconds`), then increment by `yearInSeconds = 31540000` for each school year